extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugPhotoTempData.java

```diff
@@ -0,0 +1,68 @@
+package org.datatransferproject.transfer.smugmug.photos;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import org.datatransferproject.types.common.models.DataModel;
+
+@JsonTypeName("org.dataportability:SmugMugPhotoTempData")
+public class SmugMugPhotoTempData extends DataModel implements Serializable {
+  private final String albumId;
+  private final String albumName;
+  private final String albumDescription;
+  private int photoCount;
+  private String overflowAlbumId;
+
+  @JsonCreator
+  public SmugMugPhotoTempData(
+      @JsonProperty("albumId") String albumId,
```

Comment:
export service, the one that comes from importAlbum.getId()

Reply:
The smugmug album id is not used, only the album uri is needed for uploading photos. So what ends up happening is that the importAlbum.getId() id is used to cache the album uri and to access the jobstore. The SmugMugAlbum is only used in decoding the response from the API, not stored or used in the importer. 

I wanted to include as little information in the SmugMugPhotosTempData object as possible to keep it simple and light. The name, description, and id are needed to create an overflow album from a base album. The id is also used to lookup the albumUri from the idempotentImportExecutor. The photo count is used to keep track of the number of photos in an album and the overflow id to keep track of an overflow album.

I could move the albumUri into this class and avoid the idempotentimportexecutor.getCachedValue call altogether.

Reply:
That makes sense - Since were doing this lookup for every photo album, if we can consolidate and include the album uri in the model I think that would make more sense and reduce the calls to datastore. 

Also could you change the name to make it clear that is is the export album id and not the smugmug album id?

Reply:
done and done